Made normal the select value by default

kyle46 opened the pull request:
Normal is the default difficulty in code however no default was set for the select box which then loaded the first option in the list.

Issue #192

sporchia commented:
please see if these are still issues in v8.27, most of these issues have been addressed in the upcoming release

sporchia commented:
mostly avoiding the merge conflict resolution on this one

kyle46 commented:
This issue is fixed in v8.27. I closed one of the other pull requests as it was also fixed in v8.27. The third is not yet fixed in v8.27. I switched the branch for the pull request for the other fix to v8.27. I'll make sure any pull requests I make in the future are on the latest branch rather than the master. Sorry for the confusion!
